src/equations/numerical_fluxes.jl

```diff
@@ -221,6 +221,46 @@ See [`FluxLaxFriedrichs`](@ref).
 """
 const flux_lax_friedrichs = FluxLaxFriedrichs()
 
+"""
+    DissipationEntropyStable(max_abs_speed=max_abs_speed_naive)
+
+Create a local Lax-Friedrichs-type dissipation operator that is provably entropy stable. This operator
+must be used together with an entropy-conservative two-point flux function (e.g., `flux_ec`) to yield 
```

Comment:
Does it make sense to link to an actual flux via [...](@ref) here?

Reply:
Uhmm, I don't know. I prefer to have it as general as possible. Almost no one will know what flux is `flux_ruedaramirez_etal`, but `flux_ec` is very general. BTW, there exists, in fact, a `flux_ec` in Trixi for the Burgers' equation.

Reply:
Okay, then maybe 

```md
[`flux_ec`](@ref)
```

?

Reply:
Ah I see that this is not even documented/equipped with a docstring. Maybe a good point to do this?

Reply:
I don't consider that necessary for this PR and, unfortunately, I don't have time at the moment to add that additional documentation about routines that I have not written.

Reply:
I also think that this should be left for a different PR that specifically targets documentation. This could snowball into a topic unrelated to the current task. For instance, `flux_ec` exists for Burgers' but then how should/would we "properly" document the different EC fluxes across the equation systems. What I mean is that `flux_chandrashekar` and `flux_ranocha` for the Euler equations, `flux_derigs_etal` and `flux_hindenlang_gassner` for ideal MHD, or `flux_wintermeyer_etal` and `flux_fjordholm_etal` for the shallow water equations are all EC fluxes. How one would "mask" the docstring for `flux_ec` to point to all theses disperate fluxes and equation systems is best left elsewhere.

Reply:
I can do a PR later by adding a docstring for `flux_ec` for Burgers, just to establish the link.